docs/http_api.rst

```diff
@@ -165,6 +165,66 @@ Channels
    Delete a channel.
 
 
+Logs
+^^^^
+
+.. http:get:: /channels/(channel_id:str)/logs
+
+   Get the most recent logs for a specific channel.
+
+   :query int n:
+       The amount of logs to fetch.
```

Comment:
s/amount/number/ -- maybe a bikeshed? I think "amount" is for continuous quantities. We need a grammar expert. :)